src/utils/vote-presets.js

```diff
@@ -3,8 +3,40 @@ import { erc20ABI } from 'wagmi'
 import BigNumber from 'bignumber.js'
 
 import VaultABI from './abis/Vault.json'
+import EthPNTABI from './abis/EthPNT.json'
 import settings from '../settings'
 
+const prepareInflationProposal = async (receiverAddress, rawAmount) => {
+  const Vault = new ethers.utils.Interface(VaultABI)
+  const EthPNT = new ethers.utils.Interface(EthPNTABI)
+
+  const ethPNTAsset = settings.assets.find((asset) => asset.symbol == 'ethPNT')
+  if (!ethPNTAsset) throw new Error('ethPNT asset config not found!')
+  const ethPNTAddress = ethPNTAsset.address
+  if (!ethPNTAddress) throw new Error('ethPNT asset address not found!')
+  const ethPNTDecimals = ethPNTAsset.decimals
+  if (!ethPNTDecimals) throw new Error('ethPNT asset decimals not found!')
+
+  const amount = BigNumber(rawAmount)
+    .multipliedBy(10 ** ethPNTDecimals)
+    .toFixed()
+
+  const withdrawInflationParams = [settings.contracts.financeVault, amount]
+
+  const transferParams = [ethPNTAddress, receiverAddress, amount]
+
+  return [
+    {
+      to: ethPNTAddress,
+      calldata: EthPNT.encodeFunctionData('withdrawInflation', withdrawInflationParams)
+    },
+    {
```

Comment:
Are there other cases where there are two function calls in the same proposal? In other words, does dandelion voting contract support multicall?

Reply:
If that's not the case we will be required to open two votes, one for inflating, another one for transferring.

@wernervoncroy what's your opinion here?

Reply:
It does support it apparently, the tests in the forked environment are testing this specific case